go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/docker/docker v28.3.3+incompatible
 	github.com/docker/go-units v0.5.0
 	github.com/dolmen-go/contextio v0.0.0-20200217195037-68fc5150bcd5
-	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20250918130837-16c4d9a1aa8c
+	github.com/elastic/beats/v7 v7.0.0-alpha2.0.20250924190604-3dc72030429f
```

Comment:
Can you avoid making this change in this PR? This will make it impossible to backport cleanly, as the beats dependency needs to mirror its respective branch in each agent branch.

Reply:
At some point, sooner rather than later, we should have the journald configuration updated from
```yaml
        paths:
          - "/opt/journald/"
```
to
```yaml
        paths:
          - "/opt/journald/"
```

For the sake of correctness in our tests. This is also testing a common use case of ingesting Kubernetes node logs via Journald.

Reply:
Edit, I fixed the paths in my previous comment.

Reply:
Allright, I'm fine merging it as-is. But we'll have to be careful with backports. In fact, you should set `backport-active-all` on this PR and then resolve dependencies in each backport, imo.